From acbc530666d7607df45784992d021cf1e60724d6 Mon Sep 17 00:00:00 2001
From: Samuel Jimenez <the.real.samuel.jimenez@gmail.com>
Date: Tue, 26 Mar 2024 13:38:53 -0500
Subject: [PATCH] Make warning RegEx non-greedy.

---
 src/parser/latexoutputparser.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/parser/latexoutputparser.cpp b/src/parser/latexoutputparser.cpp
index 2c2dbed42..7a0cfe6f1 100644
--- a/src/parser/latexoutputparser.cpp
+++ b/src/parser/latexoutputparser.cpp
@@ -410,7 +410,7 @@ bool LaTeXOutputParser::detectWarning(const QString & strLine, short &dwCookie)
     bool found = false, flush = false;
     QString warning;
 
-    static QRegularExpression reLaTeXWarning("^(((! )?(La|pdf)TeX)|Package|Class) .*Warning.*:(.*)", QRegularExpression::CaseInsensitiveOption);
+    static QRegularExpression reLaTeXWarning("^(((! )?(La|pdf)TeX)|Package|Class) .*Warning.*?:(.*)", QRegularExpression::CaseInsensitiveOption);
     static QRegularExpression reNoFile("No file (.*)");
     static QRegularExpression reNoAsyFile("File .* does not exist."); // FIXME can be removed when https://sourceforge.net/p/asymptote/bugs/70/ has promoted to the users
 
-- 
GitLab

