commit 37913d006d96a6bdcb963f4ca4889888dcce6094
Author: Frederik Seiffert <frederik@algoriddim.com>
Date:   Wed May 3 13:49:09 2023 +0200

    Fix GSXML compatibility with libxml2 v2.11.0
    
    Fixes #294.

diff --git a/Source/Additions/GSXML.m b/Source/Additions/GSXML.m
index 37f509564..01a24c5aa 100644
--- a/Source/Additions/GSXML.m
+++ b/Source/Additions/GSXML.m
@@ -2671,10 +2671,12 @@ getEntityDefault(void *ctx, const xmlChar *name, BOOL resolve)
                   return NULL;
                 }
               ret->owner = 1;
+#if LIBXML_VERSION < 21100
               if (ret->checked == 0)
                 {
                   ret->checked = 1;
                 }
+#endif
             }
         }
     }
